README.rst

```diff
@@ -1445,9 +1445,20 @@ Choose from the list of available rules:
     ``'newest'``
   - ``use_class_const`` (``bool``): use ::class notation; defaults to ``true``
 
-* **php_unit_ordered_covers** [@PhpCsFixer]
+* **php_unit_ordered_annotations** [@PhpCsFixer]
 
-  Order ``@covers`` annotation of PHPUnit tests.
+  Order annotations of PHPUnit tests.
+
+  Configuration options:
+
+  - ``annotations`` (a subset of ``['author', 'covers', 'dataProvider', 'depends',
```

Comment:
What about `coversNothing` and `internal`? The more the merrier in this case, right?

Reply:
ping @localheinz

Reply:
@kubawerlos 

Not sure, why would anyone have multiple

- `@coversNothing`
- `@internal`

annotations, perhaps even followed by content?

Should I add them nonetheless?

Perhaps a case for an additional fixer, remove duplicate annotations?

Reply:
@localheinz oh, I was thinking the fixer will also order the groups of annotations :man_facepalming: